src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,143 @@
+package duke;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.Arrays;
+
+/**
+ * Class for parsing user commands.
+ */
+public class Parser {
+    private static final String TODO_COMMAND = "todo";
```

Comment:
This works perfectly fine but can consider using java enums to represent these constants instead. In that way you can use switch case instead of multiple if else statements when parsing the command. You also wont have to call the .equals string comparison method in every if clause.